mdpow/fep.py

```diff
@@ -150,8 +152,16 @@
 from numkit.observables import QuantityWithError
 from glob import glob
 
+# Patch due to slightly different library names
+if sys.version_info.major == 2:
+    from ConfigParser import NoOptionError
```

Comment:
Didn't you use configparser with 2 and 3 elsewhere?

At the moment your code is not safe for Python 4. I'd write
```python
try:
  # Python 2
  import ConfigParser as configparser
except ImportError:
  import configparser
```

Removes the `import sys` and is more robust and more Pythonic.

Then use `configparser.NoOptionError` below (which also shows why I am not a fan of importing stuff into a namespace).

Reply:
better https://portingguide.readthedocs.io/en/latest/stdlib-reorg.html#renamed-modules
```python
from six.moves.configparser import ConfigParser
```